src/DynamoCoreWpf/ViewModels/Watch3D/HelixWatch3DViewModel.cs

```diff
@@ -1078,7 +1078,7 @@ private void OnSceneItemsChanged()
             lock (element3DDictionaryMutex)
             {
                 geometryModels = Element3DDictionary
-                        .Where(x => x.Key.Contains(node.AstIdentifierGuid) && x.Value as Element3D != null).ToArray();
+                        .Where(x => x.Key.Contains(node.AstIdentifierGuid) && x.Value is Element3D).ToArray();
```

Comment:
SQ has been back and forth about this, which made me think we may not need this type check at all